signcrypt_test.go

```diff
@@ -275,18 +372,18 @@ func TestSigncryptionStreamWithError(t *testing.T) {
 	sealed, err := SigncryptSeal(msg, keyring, senderSigningPrivKey, receiverBoxKeys, nil)
 	require.NoError(t, err)
 
-	// Break the final packet.
-	sealed[len(sealed)-1] ^= 1
+	// Break the final packet. (TODO: Go one back and twiddle the isFinal bit)
+	sealed[len(sealed)-2] ^= 1
```

Comment:
Is this change doing anything? Tweaking the last byte vs the second-to-last byte shouldn't make a difference, right?